Class-Examples/Python/webScrapers/grimmTales/002.txt

```diff
@@ -0,0 +1,148 @@
+Hard by a great forest dwelt a wood-cutter with his wife, who had an
+only child, a little girl three years old.  They were so poor,
+however, that they no longer had daily bread, and did not know how to
+get food for her.  One morning the wood-cutter went out sorrowfully
+to his work in the forest, and while he was cutting wood, suddenly
+there stood before him a tall and beautiful woman with a crown of
+shining stars on her head, who said to him 'I am the virgin mary,
+mother of the child jesus. You are poor and needy, bring your child
+to me, I will take her with me and be her mother, and care for her.'
```

Comment:
A suggestion for the text would be to capitalize names like Jesus and the Virgin Mary
``` to me, I will take her with me and be her mother, and care for her.'